Test/comp/Hello.dfy

```diff
@@ -2,12 +2,13 @@
 // RUN: %dafny /compile:3 /spillTargetCode:2 /compileTarget:js "%s" >> "%t"
 // RUN: %dafny /compile:3 /spillTargetCode:2 /compileTarget:go "%s" >> "%t"
 // RUN: %dafny /compile:3 /spillTargetCode:2 /compileTarget:java "%s" >> "%t"
+// RUN: %dafny /compile:3 /spillTargetCode:2 /compileTarget:py "%s" >> "%t"
 // RUN: %diff "%s.expect" "%t"
 
 method Main() {
-  print "Hello, JavaScript! Best, Dafny\n";
-  var x := 14;
-  print "x is ", x, "\n";
-  var y := false;
-  print "y is ", y, "\n";
+    print "Hello, JavaScript! Best, Dafny\n";
```

Comment:
Hello, Python!

Reply:
This is a test file, I think we should not make any changes in there.

Reply:
On the one hand I THINK Mikael was just being cheerful and silly :)

On the other hand, it would be better for a generic test run against multiple compilers not to specifically greet JavaScript and make the other languages feel somehow left out. Seems like a good change to make this "Hello, my favourite language!" while we're here! We can certainly change test cases to improve understanding or coverage.

On the third hand I'm probably overthinking the whole thing. :)

Reply:
I also think it would be better to have some string not mentioning any language.I will do it here.

Reply:
How does "Hello, World! Best, Dafny\n" sound?